packages/next/src/build/webpack-config.ts

```diff
@@ -840,6 +841,8 @@ export default async function getBaseWebpackConfig(
     },
   }
 
+  const aliasCodeConditionTest = [codeCondition.test, pageExtensionsRegex]
```

Comment:
Makes sense but why calling it alias-?

Reply:
`codeCodition.test` are available for both webpack **alias** and **transform** rules, I created this one to imply that we only want to apply the webpack aliases for this group.